pym/bob/cmds/build.py

```diff
@@ -333,6 +333,8 @@ def _generateAudit(self, step, depth, resultHash):
         audit = Audit.create(step.getVariantId(), self._getBuildId(step, depth), resultHash)
         audit.addDefine("bob", BOB_VERSION)
         audit.addDefine("recipe", step.getPackage().getRecipe().getName())
+        for var,val in step.getPackage().getMetaVars().items():
+            audit.addDefine(var, val)
```

Comment:
hmm...I don't think the existing "build" and "meta" namespaces should be merged. But in the audit there is also "env" --> "metaEnv" as a new namespace ?